index.php

```diff
@@ -0,0 +1,273 @@
+<!-- PHP Logic starts here to fetch info  -->
+<?php
+
+  /* Include the User class file. */
+  require_once 'user.php';
+
+  if ($_SERVER["REQUEST_METHOD"] == "POST") {
+    /**
+     * Function to remove extra spaces,
+     * backslashes and convert special characters
+     * to HTML entites form user input data.
+     *
+     * @param string $data
+     *
+     * @return string $data
+     *
+     */
+    function checkInput($data) {
+      //Removes extra spaces from the begning and end.
+      $data = trim($data);
+      // Removes backslashes.
+      $data = stripslashes($data);
+      // convert special characters to HTML entities.
+      $data = htmlspecialchars($data);
+      return $data;
+    }
+    //Regex Pattern.
+    $regex = "/^[a-zA-Z-\s' ]*$/";
+    $numRegex = "/^[0-9]+$/";
+    $emailErr = $fnameErr = $lnameErr = $numErr = $emailSuccess = $emailMsg = "";
+
+    /**
+     * Checking user inputs.
+     *
+     * @param string $input
+     * Input from user.
+     *
+     * @param string $preg
+     * Regex to match pattern.
+     *
+     * @param &string $error
+     * Address of error @var to update.
+     *
+     * @param string $fieldName
+     * Field name of the input.
+     */
+
+    function validate($input, $preg, &$error, $fieldName) {
+      //Checking empty input.
+      if (empty ($input)) {
+        $error = "* $fieldName Name is required";
+        return "";
+      }
+      else {
+        $name = checkInput($input);
+        //Checking to Pattern from regex.
+        if (!preg_match($preg, $name)) {
+          $error = "* Invalid Input Type.";
+          return "";
+        }
+        //Checking empty @var name.
+        else if (empty ($name)) {
+          $error = "* Empty, Please enter valid $fieldName .";
+          return "";
+        }
+        else {
+          return $input;
+        }
+      }
+    }
+
+    /**
+     * Function checks email validation.
+     *
+     * @param string $email
+     * Input from user.
+     * @param string $emailErr
+     *
+     * @param string $emailSuccess
+     *
+     * @param string $emailMsg
+     *
+     * @return string $email
+     */
+
+    function validateEmail($email, &$emailErr, &$emailSuccess, &$emailMsg) {
+      //Check empty $email.
+      if (empty ($email)) {
+        $emailErr = "* Email is required";
+        return "";
+        }
+        else {
+          $email = checkInput($email);
+          if (!filter_var($email, FILTER_VALIDATE_EMAIL)) {
+            $emailErr = "* Invalid email format.";
+            return "";
+          }
+          else {
+            // Initialize a curl session.
+            $ch = curl_init();
+            //Set curl options.
+            curl_setopt($ch, CURLOPT_URL, 'https://emailvalidation.abstractapi.com/v1/?api_key=88e808712da846bf889378a04dd4d4f0&email=' . $email);
```

Comment:
We should not push `api_key` or any other credentials in the GitHub. Please use `.env` file.